user-guide/Advanced_Modules/User-Defined_APIs/Defining_an_API/UD_APIs_Defining_a_new_API.md

```diff
@@ -151,7 +151,7 @@ The `ApiTriggerOutput` object has the following properties:
 | Property | Type | Description |
 |--|--|--|
 | ResponseCode | int | The HTTP status code. See [ResponseCode](#responsecode). |
-| ResponseBody | string | Contains the response body as a string. |
+| ResponseBody | string | Contains the response body as a string. The size of this response body is limited to 29 MB from DataMiner 10.4.3/10.5.0 onwards. |
```

Comment:
Maybe we can remove the 'from version' part as it was previously 30 without a specific error. It would be clearer imo that this limit is kept in mind for all versions, as this may suggest that you can go way beyond this in previous versions.

Reply:
Done